src/i18n/scripts/intl-imports.js

```diff
@@ -0,0 +1,260 @@
+#!/usr/bin/env node
+
+const scriptHelpDocument = `
+NAME
+  intl-imports.js —  Script to generate 'index.js' files for Micro-front-ends.
```

Comment:
Clarity nit:

> Script to generate 'index.js' files for Micro-front-ends.

This feels like a bit too generic a statement - index.js files are used all over the place to present exports from modules.  Might want to say something like

"Script to generate an i18n index.js file that exports messages from all the languages present in the i18n/messages folder."

Something like that.

Reply:
Done, thanks :). Doc updates are always great!!